modules/API/pages/index.adoc

```diff
@@ -0,0 +1,41 @@
+= GSQL SERVER REST API
+:description: Overview of GSQL SERVER's REST API.
+:pp: {plus}{plus}
+:page-aliases: API:intro.adoc, api:intro.adoc
+
+This user guide provides information on how to engage with our REST APIs: the introduction section explains how to send requests, pass parameters, and format request bodies, while xref:gsql-endpoints.adoc[] describes in detail each endpoint and its input and output.
+
+To submit a request, send an HTTP request to the NGINX server or the GSQL server.
+By default, the GSQL server listens for requests at port 8123 and the NGINX server listens on port 14240.
+A request needs to specify the following:

```

Comment:
Let's rephrase to a much clearer sentence: "A request must include the following specifications:"